configuration/driver/icedrv_flux.F90

```diff
@@ -272,8 +273,6 @@ module icedrv_flux
       !-----------------------------------------------------------------
 
       real (kind=dbl_kind), dimension (nx), public :: &
-         rside   , & ! fraction of ice that melts laterally
```

Comment:
wouldn't we want to keep the average value as a diagnostic?

Reply:
We didn't have rside as a diagnostic before, but we could add it.